src/Bridge/Twig/Extension/EnumExtension.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Greg0ire\Enum\Bridge\Twig\Extension;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+final class EnumExtension extends \Twig_Extension
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getFunctions()
+    {
+        return [
+            new \Twig_SimpleFunction('enum_label', [$this, 'label']),
+        ];
+    }
+
+    /**
```

Comment:
1. Create an annotation (or reuse the one from the validator)
2. Create a class that will parse metadata (or reuse one)
3. Create a class that will be a glorified associative hash `Class::Field` => `EnumClass`
4. Inject this class in the twig extension
5. Profit